t/test.c

```diff
@@ -569,6 +569,73 @@ static void test_nondecryptable_initial(void)
 #undef LEN_LOW
 }
 
+static void test_quicly_set_cc(void)
```

Comment:
```suggestion
static void test_set_cc(void)
```

All the tests are about quicly.